### DISCUSSION TITLE
F-Droid

### DISCUSSION THREAD

**Discussion by hashlobster** [Mygod/VPNHotspot#209] [Show and tell]
Hello,

I have decided to maintain a [FOSS fork](https://github.com/hashlobster/VPNHotspot-FOSS) to be published on F-droid again.

Just wanted to inform you once before submitting it there.

**Comment by Mygod** [comment]
Looks good (thanks for changing the package name). You might want to make the repo so that it is a fork of this repo?

**Comment by Mygod** [comment]
By the way, I don't think `com.android.billingclient:billing-ktx` is not F-droid complaint.
